.github/stale.yml

```diff
@@ -11,16 +13,26 @@ exemptLabels:
   - feature
   - possible_feature
   - fix-next
-  - itext
+  - text
   - low_priority
   - svg_output
   - svg_parser
+  - v6
+  - group
+  - task
+  - help wanted
+  - CI/CD
+  - experimental
 # Label to use when marking an issue as stale
 staleLabel: stale
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: true
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
   recent activity. It will be closed if no further activity occurs. Thank you
   for your contributions.
 # Comment to post when closing a stale issue. Set to `false` to disable
 closeComment: false
+# Limit to only `issues` or `pulls`
+only: issues
```

Comment:
PR and Issues should be bot triggered. If someone open a PR and disappear, doesn't give feedback even if asked, a stale label is fine

Reply:
do you mind we give it 30 days instead of 14 for issues?

Reply:
c642a00